### ISSUE TITLE
AVSBench-Object Code Release

### ISSUE THREAD

**Issue by Shui13ai** [cyh-0/CAVP#3]
Hi, it seems that only the code for AVSBench-Semantic has been released. Could you please also release the code for AVSBench-Object? Thank you!

**Comment by cyh-0** [comment]
Hi @Shui13ai ,

The code related to AVSBench-Object is relate to [main_avss_resize.py](https://github.com/cyh-0/CAVP/blob/main/main_avss_resize.py)

**Comment by Shui13ai** [comment]
Hi@cyh-0,
I think main_avss_resize.py still uses AVSSDataset instead of S4 setting according to line 156. Could you please recheck it?
https://github.com/cyh-0/CAVP/blob/main/main_avss_resize.py#L156

**Comment by cyh-0** [comment]
I wrap the binarise process inside the AVSSDataset https://github.com/cyh-0/CAVP/blob/d9c0a2b4d088d5c63f5962d18c85136fc34a4232/dataset/avss/visual/visual_dataset.py#L156-L158 have updated the bash script, we can use the following command to train on the specific split.
```
python main_avss_resize.py --experiment_name "CAVP" --setup avss --avsbench_split "v1s" --gpus 1 --batch_size 16 --lr 1e-3 --weight_decay 1e-4 --epochs 60 --wandb_mode disabled --num_workers 16
```

**Comment by Shui13ai** [comment]
@cyh-0 
Hi, when using the command above, the Loss stopped reducing, and the mIoU remained the same after a few epochs. After checking the code again, it seems that main_avss_resize.py is still using the same trainer. Therefore, it is calculating CrossEntropy Loss with binary labels. Should I change the loss function when running on v1s split?( https://github.com/cyh-0/CAVP/blob/main/trainer/trainer_cavp_avss_image.py#L218 )
Besides, it seems that the mIoU is calculated according to num_classes=71, yet we have only 2 classes when training on v1s split. I'm wondering if I should change the calculation of mIoU as well? ( https://github.com/cyh-0/CAVP/blob/main/utils/eval_utils.py#L33 )

**Comment by cyh-0** [comment]
Sorry, I miss the resize_flag and the setup string was incorrect:

python main_avss_resize.py --experiment_name "CAVP" --setup avss_binary --resize_flag --avsbench_split "v1s" --gpus 1 --batch_size 16 --lr 1e-3 --weight_decay 1e-4 --epochs 60 --wandb_mode disabled --num_workers 16

**Comment by Shui13ai** [comment]
It works well now. Thank you for the correction!
